### PR TITLE
Hello rails react

### PR DESCRIPTION
**Pull request by iLynette**
## Project requirements

- Create a new Rails app called 'hello-rails-react'.

- Initialize your project with Git.

- Set up your project with webpacker the react-rails gems as describe in the previous lesson's tutorial.

- Make sure that your project has a Postgres database set up.

- Create a table for storing your messages and create 5 different greetings.

- Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).

- Create a static view that will be the root of your app.

- Create the App component with react-router and render it in your static view.

- Create the Greeting component that will display a greeting. Set it up as a route in your App component.

- Create a store, an action and a reducer that will connect to you API endpont to get the random greeting.

- Display the random greeting in your Greeting component.

### CODE REVIEW

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @iLynette, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

175083599-879bf673-fb93-406f-939c-de7e586bf6d9

## Highlights 👏 
✅ Random greetings are displayed on refresh 🔥 
✅ App has the correct functionality 👍 
✅ Good commit history 👏 
✅ Linters are passing 👌 


## Required Changes ♻️
- [ ] Kindly edit your README file using this [Microverse template](https://github.com/microverseinc/readme-template) with your information and all the necessary commands which include `bundle i`, `npm i`, `rails webpacker:install`, `rails webpacker:install:react`, `rails db:create db:migrate` that a user would need to set up and run your project locally 💯 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,24 +1,87 @@
-# README
+# Budget App
```
Comment (RolandM99) [inline README.md]: - [ ] Kindly, change your project title in your Readme file.

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @iLynette, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (iLynette) [reply README.md]: Thank you @RolandM99 @Kingstalux I've fixed the README

**Pull request merged**